drivers/pinctrl/pinctrl_rv32m1.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022 Henrik Brix Andersen <henrik@brixandersen.dk>
+ * Copyright (c) 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT openisa_rv32m1_pinmux
+
+#include <drivers/pinctrl.h>
```

Comment:
`<zephyr/...>` (I wonder who thought it was a good idea to allow prefixed/non-prefixed includes upstream)

Reply:
Fixed.